### PR TITLE
Add CONTRIBUTING.MD and update RELEASES.md

### PR DESCRIPTION
**Pull request by Gankra**

### CODE REVIEW

**Comment by Gankra** [comment]
[rendered](https://github.com/Gankra/rust-minidump/blob/contrib/CONTRIBUTING.md)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+
+    assert!(output.status.success());
+    assert_eq!(stderr, "");
+
+    // Hey insta, did the result change?
+    insta::assert_snapshot!("json-pretty-evil-symbols", stdout);
+}
+```
+
+Part of the motivation for this is to ensure we don't break the the JSON output, which has [very detailed schema document](https://github.com/luser/rust-minidump/blob/40c3390f5705890f932f78b7db4fc02866e012b8/minidump-processor/json-schema.md), which we'retrying to keep stable so people can actually rely on it while the actual implementation details are still in flux.
```
Comment (5225225) [inline CONTRIBUTING.md:290]: > [...] which **we'retrying** to [...]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+
+Major locations for tests include:
+
+* [minidump parsing unit tests](https://github.com/luser/rust-minidump/blob/2001547fcf4aa0f28f52b8b1ab5da9bd99c8ac87/minidump/src/minidump.rs#L4739) (see the `synth-minidump` section)
+* [minidump-processor integration tests](https://github.com/luser/rust-minidump/blob/master/minidump-processor/tests/test_processor.rs) (see the `synth-minidump` section)
+* [minidump-stackwalk integration tests](https://github.com/luser/rust-minidump/blob/master/minidump-stackwalk/tests/test-minidump-stackwalk.rs) (see the `insta` section)
+* [breakpad-symbols symbol file parser tests](https://github.com/luser/rust-minidump/blob/2001547fcf4aa0f28f52b8b1ab5da9bd99c8ac87/breakpad-symbols/src/lib.rs#L819)
+* [breakpad-symbols cfi interpretter tests](https://github.com/luser/rust-minidump/blob/master/breakpad-symbols/src/sym_file/walker.rs#L1032)
```
Comment (5225225) [inline CONTRIBUTING.md:94]: is interpretter a typo here?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+Oh also if **test_minidump_stackwalk__markdown-help.snap changes, use the new contents to update minidump-stackwalk's README.md**.
+
+
+
+
+## cargo fuzz: Fuzzing The Project
+
+Many of the crates have fuzzing tests. These tests aren't run in CI, but CI does check that they continue to compile. As of this writing, the fuzzer used does not work on Windows (sorry!).
+
+You can each fuzzer with `cargo fuzz`, but sadly this command is not workspace aware, so you will have to `cd` into the particular project you want to fuzz.
```
Comment (5225225) [inline CONTRIBUTING.md:303]: "You can *run* each fuzzer"?

**Pull request merged**